src/WingmanDataTrail/MetricsList/SimpleMetricsList.tsx

```diff
@@ -27,6 +27,8 @@ import {
   MetricVizPanel,
 } from 'WingmanDataTrail/MetricVizPanel/MetricVizPanel';
 import { SceneByVariableRepeater } from 'WingmanDataTrail/SceneByVariableRepeater/SceneByVariableRepeater';
+import { VAR_FILTERS } from 'shared';
+import { isAdHocFiltersVariable } from 'utils/utils.variables';
```

Comment:
I'm getting a lint error with the order of these imports:
```
  30:1  error  `shared` import should occur before import of `utils`                                                        import/order
  31:1  error  `utils/utils.variables` import should occur before import of `WingmanDataTrail/ListControls/LayoutSwitcher`  import/order
```

It's fixed via:

```
npm run lint:fix
```

Reply:
fixed in https://github.com/grafana/metrics-drilldown/pull/361/commits/b616ff9aa2a848eff5911c5f544a4cca67448eec